pkg/codegen/buildconfig/main.go

```diff
@@ -1,3 +1,5 @@
+//go:debug gotypesalias=0
```

Comment:
Would you mind adding a comment on why the directive is there and when it can be removed?

Reply:
Same for pkg/codegen/main.go